mobile/lib/services/location_service.dart

```diff
@@ -199,10 +199,9 @@ class LocationService {
       latitude,
       longitude,
     );
+    nearestSites.sortByDistanceToReferenceSite();
 
```

Comment:
I would like to appreciate why the nearest location keeps on changing rapidly on my phone. Could it be that the operation keeps on refreshing in the background? @NoahNsimbe

Reply:
Are we able to keep it stable (same place) for sometime? Just need to be enlightened. Nothing much really.

Reply:
Thanks @Baalmart this PR is a setup for tests. We shall address nearest location in a separate PR.

Reply:
cool